### ISSUE TITLE
CIFS not recognised

### ISSUE THREAD

**Issue by tellytart** [echocat/nagios-plugin-check_mountpoints#32]
I'm trying this script on CentOS7.1, and while it's working correctly with one server, but not another. On investigation on the non-working server the CIFS mounts had trailing /'s on both the source and mountpoint. Removing the trailing / fixed the issue - but perhaps the script can catch this and not error?

**Comment by dwerder** [comment]
Do you use the latest version? Because the regex already ignores the last / 
```
MPS=`${GREP} -v '^#' ${FSTAB} | awk '{if ('${NOAUTOCOND}'&&($'${FSF}'=="ext3" || $'${FSF}'=="xfs" || $'${FSF}'=="auto" || $'${FSF}'=="ext4" || $'${FSF}'=="nfs" || $'${FSF}'=="nfs4" || $'${FSF}'=="davfs" || $'${FSF}'=="cifs" || $'${FSF}'=="fuse" || $'${FSF}'=="glusterfs" || $'${FSF}'=="ocfs2" || $'${FSF}'=="lustre" || $'${FSF}'=="ufs" || $'${FSF}'=="zfs"))print $'${MF}'}' | tr '\n' ' ' | sed -e 's/\/$//i'`
```

This "sed" removes the tailing slash

```
sed -e 's/\/$//i'
```

**Comment by tellytart** [comment]
Yes, I grabbed the latest version from the GitHub. As soon as I removed the trailing slashes in fstsb the script worked fine.

**Comment by dwerder** [comment]
Fixed by a2dd8db1f4eac2b858476c350f4922370651dd58
